Pending_updates/wat_026_wastewater_treatment_plants/wat_026_wastewater_treatment_plants_processing.py

```diff
@@ -0,0 +1,191 @@
+import geopandas as gpd
+import glob
+import requests
+import os
+import urllib.request
+import sys
+from collections import OrderedDict
+import cartosql
+from carto.datasets import DatasetManager
+from carto.auth import APIKeyAuthClient
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+import util_files
+import util_cloud
+from zipfile import ZipFile
+import logging
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of table on Carto where you want to upload data
+# this should be a table name that is not currently in use
+dataset_name = 'wat_026_wastewater_treatment_plants' #check
+
+logger.info('Executing script for dataset: ' + dataset_name)
+# create a new sub-directory within your specified dir called 'data'
+# within this directory, create files to store raw and processed data
+data_dir = util_files.prep_dirs(dataset_name)
+
+'''
+Download data and save to your data directory
+'''
+# insert the url used to download the data from the source website
+url = 'https://opendata.arcgis.com/datasets/4b9bac25263047c19e617d7bd7b30701_0.zip?outSR=%7B%22latestWkid%22%3A3857%2C%22wkid%22%3A102100%7D' #check
+
+# download the data from the source
+r = requests.get(url)
+raw_data_file = os.path.join(data_dir, 'Environmental_Protection_Agency__EPA__Facility_Registry_Service__FRS__Wastewater_Treatment_Plants-shp.zip')
+urllib.request.urlretrieve(url, raw_data_file)
+
+# unzip source data
+raw_data_file_unzipped = os.path.join(data_dir, 'unzip')
+zip_ref = ZipFile(raw_data_file, 'r')
+zip_ref.extractall(raw_data_file_unzipped)
+zip_ref.close()
+
+'''
+Process data
+'''
+# load in the polygon shapefile
+shapefile = glob.glob(os.path.join(raw_data_file_unzipped, '*.shp'))[0]
+gdf = gpd.read_file(shapefile)
+
+# Reproject geometries to epsg 4326
```

Comment:
just want to make sure it is noted that this is WGS84 because that's what it says in the README (in case a user doesn't realize these are the same thing)
```suggestion
# Reproject geometries to WGS84 (epsg 4326)
```